src/nn/ops/conv2d.ts

```diff
@@ -14,8 +14,7 @@ export class Conv2d extends SingleOutputOperation implements FusedOperation {
   private input_: MLOperand;
   private filter_: MLOperand;
   private bias_: MLOperand;
-  private padding_?: [number, number, number, number];
-  private tfPadding_: ExplicitPadding;
+  private padding_?: ExplicitPadding;
```

Comment:
I didn't mean rename. I meant use `padding_` for output shape inference while use `tfPadding_` for tf.js interaction.

Reply:
Updated. Please review this update in new commit, thanks.